src/kernel/capinv.c

```diff
@@ -1531,6 +1550,19 @@ composite_syscall_slowpath(struct pt_regs *regs, int *thd_switch)
 
 			break;
 		}
+		case CAPTBL_OP_TCAP_WAKEUP:
+		{
+			struct cap_tcap *tcapwkup = (struct cap_tcap *)ch;
```

Comment:
It doesn't say much about variables, aside from the convention to not use verbs first (which was broken earlier, but for a good reason, I believe).  We aren't going full hungarian here, just encouraging good use of variable names.  I don't have a strong preference personally for `_` or not here.